internal/storage/memory/memory_test.go

```diff
@@ -149,13 +149,13 @@ func Benchmark_Storage_Memory(b *testing.B) {
 		b.ResetTimer()
 		for n := 0; n < b.N; n++ {
 			for _, key := range keys {
-				_ = d.Set(key, value, ttl)
+				d.Set(key, value, ttl)
```

Comment:
Doesn't it return an error here? I prefer it to be explicit that you ignore returns using underline because it stays readable and creates curiosity when to analysis tests.